module/Education/view/education/education/index.phtml

```diff
@@ -114,6 +114,13 @@ $this->headTitle($this->translate('Education'));
                     <div class="panel-heading">
                         <span class="h4"><?= $this->translate('Books') ?></span>
                     </div>
+                    <div class="panel-thumb">
+                        <a class="thumbnail" href="https://mindbus.go2cloud.org/aff_c?offer_id=65&aff_id=639&url=https://www.athenaeum.nl/studieboeken/studieverenigingen/?utm_source=639" style="margin-bottom: 0 !important;">
```

Comment:
Shouldn't we just make this refer to https://gewis.nl/books. The CBC has already adjusted this link to this one.

Reply:
@Koen1999 I suppose we can do that. I did it this way because this is "as is".

Reply:
Mmmmhm, also seems reasonable. We'll do the easy way from here